taxcalc/calcfunctions.py

```diff
@@ -401,6 +401,49 @@ def AGI(ymod1, c02500, c02900, XTOT, MARS, sep, DSI, exact, nu18, taxable_ubi,
     """
     Computes Adjusted Gross Income (AGI), c00100, and
     compute personal exemption amount, c04600.
+
+    Parameters
+    ----------
+    ymod1: float
+
+    c02500: float
+
+    c02900: float
+
+    XTOT: float
+
+    MARS: int
+        filing status
+    sep: float
+
+    DSI: float
+
+    exact: int
+        exact == 1 means do exact calculation, else do smoothed calculation
+        which is used for marginal tax rates
+    nu18: int
+        number of dependents under age 18
+    taxable_ubi: float
+        taxable UBI amount
+    II_em: 
```

Comment:
Just an idea that occurred to me in passing: I'm not sure 100% how it would work, but it'd be neat to pull these directly from `policy_current_law.json`. We could probably do this with the records variables, too: https://github.com/PSLmodels/Tax-Calculator/blob/master/taxcalc/records_variables.json

Reply:
@hdoupe Do you mean the description for the docstring?

Reply:
Ah, yes that's what I was thinking about.